README.md

```diff
@@ -6,28 +6,19 @@
 
 _[Ably](https://ably.com) is the platform that powers synchronized digital experiences in realtime. Whether attending an event in a virtual venue, receiving realtime financial information, or monitoring live car performance data – consumers simply expect realtime digital experiences as standard. Ably provides a suite of APIs to build, extend, and deliver powerful digital experiences in realtime for more than 250 million devices across 80 countries each month. Organizations like Bloomberg, HubSpot, Verizon, and Hopin depend on Ably’s platform to offload the growing complexity of business-critical realtime data synchronization at global scale. For more information, see the [Ably documentation](https://ably.com/documentation)._
 
-This is a Go client library for Ably.
+## Overview
+This is a Go client library for Ably. [Supported features](#feature-support) and [known limitations](#known-limitations) are documented here.
 
 ## Installation
 
 ```bash
 ~ $ go get -u github.com/ably/ably-go/ably
 ```
 
```

Comment:
The template says the installation section should provide a link to the requirements section
https://github.com/ably/ably-common/blob/main/templates/sdk-readme.md#installation

So I think we need something like this in the installation section
```
See [Requirements](#requirements)
```